packages/web/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "compileOnSave": false,
```

Comment:
can you extend our core shared file here or no?

Reply:
hmm not without having to change code to fix a lot of new compile errors